tools/Propulsion.Tool/Program.fs

```diff
@@ -377,6 +379,7 @@ let main argv =
             let c = Args.Configuration(Environment.GetEnvironmentVariable >> Option.ofObj)
             try match a.GetSubCommand() with
                 | Init a ->         CosmosInit.aux (c, a) |> Async.Ignore<Microsoft.Azure.Cosmos.Container> |> Async.RunSynchronously
+                | InitPg a ->       MessageDb.Arguments(c, a).CreateCheckpointStore() |> Async.RunSynchronously
                 | Checkpoint a ->   Checkpoints.readOrOverride (c, a) |> Async.RunSynchronously
                 | Index a ->        Indexer.run (c, a) |> Async.RunSynchronously
                 | Project a ->      Project.run (c, a) |> Async.RunSynchronously
```

Comment:
Could wire in support for `propulsion project stats mdb` by adding an `mdb` argument to StatsParameters + KafkaParameters (only if you find it interesting)

Reply:
Definitely interesting, makes it easy to test whether it works 😄. It's wired for kafka and stats now